rosbag2_cpp/include/rosbag2_cpp/storage_options.hpp

```diff
@@ -31,6 +31,13 @@ struct StorageOptions
    * A value of 0 indicates that bagfile splitting will not be used.
    */
   uint64_t max_bagfile_size;
+  /**
+   * The chunk_size value is used to control writes to sqlite db,
+   * Selecting a optimal chunk_size, optimizes the sqlite writes by
+   * enabling transaction's
+   */
```

Comment:
two things: 
* please adhere to the existing style in this file. Add a new line after `uint64_t max_bagfile_size;` and the comment as well as remove the extra line after the comment.
* keep in mind that the `chunk_size` is independent of the actual plugin being used as there are many of them. So please remove the notion of `sqlite` and `transaction`. Same below.